lint.swift

```diff
@@ -0,0 +1,547 @@
+    try Lint.checkFileContents(
+        checkInfo: "LateForceUnwrapping3: Don't use ? first to force unwrap later – directly unwrap within the parantheses.",
+        regex: [
+            "openingBrace": #"\("#,
+            "callPart1": #"[^\s\?\.]+"#,
+            "separator1": #"\?\."#,
+            "callPart2": #"[^\s\?\.]+"#,
+            "separator2": #"\?\."#,
+            "callPart3": #"[^\s\?\.]+"#,
+            "separator3": #"\?\."#,
+            "callPart4": #"[^\s\?\.]+"#,
+            "closingBraceUnwrap": #"\)!"#
+        ],
+        matchingExamples: ["let x = (viewModel?.user?.profile?.imagePath)!\n"],
+        nonMatchingExamples: ["call(x: (viewModel?.username)!)", "let x = viewModel!.user!.profile!.imagePath\n"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: "$callPart1!.$callPart2!.$callPart3!.$callPart4",
+        autoCorrectExamples: [
+            ["before": "let x = (viewModel?.user?.profile?.imagePath)!\n", "after": "let x = viewModel!.user!.profile!.imagePath\n"]
+        ]
+    )
+
+    // MARK: LateForceUnwrapping2
+    try Lint.checkFileContents(
+        checkInfo: "LateForceUnwrapping2: Don't use ? first to force unwrap later – directly unwrap within the parantheses.",
+        regex: [
+            "openingBrace": #"\("#,
+            "callPart1": #"[^\s\?\.]+"#,
+            "separator1": #"\?\."#,
+            "callPart2": #"[^\s\?\.]+"#,
+            "separator2": #"\?\."#,
+            "callPart3": #"[^\s\?\.]+"#,
+            "closingBraceUnwrap": #"\)!"#
+        ],
+        matchingExamples: ["call(x: (viewModel?.profile?.username)!)"],
+        nonMatchingExamples: ["let x = (viewModel?.user?.profile?.imagePath)!\n", "let x = viewModel!.profile!.imagePath\n"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: "$callPart1!.$callPart2!.$callPart3",
+        autoCorrectExamples: [
+            ["before": "let x = (viewModel?.profile?.imagePath)!\n", "after": "let x = viewModel!.profile!.imagePath\n"]
+        ]
+    )
+
+    // MARK: LateForceUnwrapping1
+    try Lint.checkFileContents(
+        checkInfo: "LateForceUnwrapping1: Don't use ? first to force unwrap later – directly unwrap within the parantheses.",
+        regex: [
+            "openingBrace": #"\("#,
+            "callPart1": #"[^\s\?\.]+"#,
+            "separator1": #"\?\."#,
+            "callPart2": #"[^\s\?\.]+"#,
+            "closingBraceUnwrap": #"\)!"#
+        ],
+        matchingExamples: ["call(x: (viewModel?.username)!)"],
+        nonMatchingExamples: ["call(x: (viewModel?.profile?.username)!)", "call(x: viewModel!.username)"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: "$callPart1!.$callPart2",
+        autoCorrectExamples: [
+            ["before": "call(x: (viewModel?.username)!)", "after": "call(x: viewModel!.username)"]
```

Comment:
Codacy found an issue: [Multi-line collection literals should have trailing commas.](https://app.codacy.com/gh/Flinesoft/HandySwift/pullRequest?prid=5730214)